test/client.test.js

```diff
@@ -0,0 +1,18 @@
+import test from 'blue-tape'
+import { forEach, toPairs } from 'ramda'
+import * as c from '../src/client'
+
+test('browserLanguage', t => {
+  const testStrings = {
+    cs: ['cs', 'cs-CZ', 'CS_Czech'],
+    en: ['en', 'en-US', 'American English', 'foo', false, null],
+  }
+
+  forEach(([lang, strings]) => {
+    forEach(string => {
+      t.equal(c.browserLanguage(string), lang, `returns ${lang} on browser lang ${string}`)
```

Comment:
- Assign output of browserLanguage to `actual` const
- rename `lang` to `expectedLang`
- rename `string` to `testStr`
!fix